site/_docs/howto.md

```diff
@@ -684,8 +688,13 @@ Note: release artifacts (dist.apache.org and repository.apache.org) are managed
 
 Before you start:
 
+* Send an email to [dev@calcite.apache.org](mailto:dev@calcite.apache.org) notifying that RC build process
+  is starting and therefore `master` branch is in code freeze until further notice.
 * Set up signing keys as described above.
 * Make sure you are using JDK 8 (not 9 or 10).
+* Make sure `master` branch and `site` branch are in sync, i.e. there is no commit on `site` that has not
+  been applied also to `master`. You can check by doing `git checkout site; git rebase -i master`.
+  If they are not in sync, cherry-pick the missing commit(s) into `master`.
```

Comment:
> If they are not in sync, cherry-pick the missing commit(s) into `master`.

This is exactly what `git checkout site; git rebase -i master` does, so it might be misleading to recommend `rebase -i master` and then suggest to cherry-pick.

Reply:
You're right, I will re-phrase it.

Reply:
It would be nice to publish documentation for all the versions (all released and the current development), then we could drop `site` branch altogether :)

Reply:
Agree, there was a discussion about that some time ago [1], but I think no action has been taken in that direction.

[1] https://lists.apache.org/thread.html/0d9829f7e32f51bc03fc350fe7c782c03dedb2ecca90e983917abf53%40%3Cdev.calcite.apache.org%3E

Reply:
@vlsi correct me if I'm wrong. The goal of this point is to ensure that "there is no commit on `site` that has not been applied also to `master`", otherwise we should apply these commits (onto `master`).
Doing a `git checkout site; git rebase -i master` will point the commits on `site` that are not in `master` (if any); but will not modify the `master` branch. That is why the `cherry-pick` tried to express; or am I wrong?

Reply:
You are right. Something like `git checkout master; git reset --hard site` is required to update master to the "new site".

Reply:
We need to be careful because there can be site-related commits in `master` that are not in `site`, this is "allowed" and we must not lose them.
Having commits in `site` that are not in `master` should not happen, but it can be possible if someone makes a (small) mistake.
All in all, I think the current phrase (`git checkout site; git rebase -i master` + `cherry-pick` any missing commits into master) might be the safest approach to achieve this without losing any master commit. wdyt?

Reply:
AFAIK `git switch site; git rebase --empty=drop master; git switch master; git reset --hard site` should so the trick.

Frankly speaking, the key problem for me was to understand the intention behind `site` vs `master` branch. The actual commands are not that hard once you understand why `site` and `master` can differ.

Reply:
Ok, rephrased. I'll trust you on this git magic :)

Reply:
Oh, I guess it would be better to use `git switch site && git rebase --empty=drop master && git switch master && git reset --hard site` to avoid accidental `git reset --hard` in case one of the previous commands fail :)

Reply:
OK, changed.